Sources/XCTest/XCTestSuite.swift

```diff
@@ -68,3 +68,33 @@ public class XCTestSuite: XCTest {
         tests.append(test)
     }
 }
+
+/// A test suite which is associated with a particular test case class. It will
+/// call `setUp` and `tearDown` on the class itself before and after invoking
+/// all of the test cases making up the class.
+internal class XCTestCaseSuite: XCTestSuite {
+    private let testCaseClass: XCTestCase.Type?
+
+    init(testCaseEntry: XCTestCaseEntry) {
+        let testCaseClass = testCaseEntry.0
```

Comment:
Well have a look at that..the tuple actually already had named attributes and I just neglected to take advantage of them here. Oops!